src/backend/gl/src/queue.rs

```diff
@@ -987,7 +987,7 @@ impl Queue {
                     _ => unsafe { gl.disable(gl_offset) },
                 }
 
-                if !self.share.info.is_webgl() && !self.share.info.version.is_embedded {
+                if !crate::is_webgl() && !self.share.info.version.is_embedded {
```

Comment:
looks like `is_webgl` here is redundant. If it's not ES, then it's not WebGL for sure